Cargo.toml

```diff
@@ -19,7 +19,7 @@ futures = "0.3"
 libc = "0.2.138"
 vsock = "0.5.1"
 tokio = { version = "1", features = ["net", "sync"] }
-tonic = { version = "0.12.0", optional = true }
+tonic = { version = ">= 0.5.0", optional = true }
```

Comment:
Please restrict the upper version to `0.12.0`, we don't want to unintentionally break others if `0.13.0` or a future version introduces a breaking change to the `Connect` trait.

I'd rather manually bump it.

Reply:
No problem. Let me update it